llvm/test/tools/llvm-readobj/ELF/hex-dump-multi.s

```diff
@@ -1,10 +1,12 @@
 # REQUIRES: x86-registered-target
 
 # RUN: llvm-mc -filetype=obj -triple x86_64 %s -o %t.o
-# RUN: llvm-readobj -x .a -x .b %t.o | FileCheck %s
+# RUN: llvm-readobj -x .a -x .b %t.o | FileCheck %s --check-prefixes=HEADER,CHECK
 # RUN: llvm-readelf -x .a -x .b %t.o | FileCheck %s
 
-# CHECK:      Hex dump of section '.a':
+# HEADER:     LoadName:
```

Comment:
Is the HEADER check here actually important to the test? It feels to me like you could do the same as in hex-dump.s and just check the blank line.

Reply:
`llvm-readobj -x .a -x .b` prints two blank lines before the first `Hex dump of section`.
We have to test `LoadName:` to skip the first blank line (not due to hex/string dump).